### PR TITLE
Fix registry DNS lookup problem

### PR DESCRIPTION
**Pull request by cs-zhang**

### CODE REVIEW

File: templates/zonefile.j2 [hunk 1]
```diff
@@ -23,7 +23,7 @@ api-int		IN	A	{{ helper.ipaddr }}
 ;
 {%if setup_registry %}
 ; Create entry for the local registry
-{{ registry_host }}	IN	A	{{ helper.ipaddr }}
+registry	IN	A	{{ helper.ipaddr }}
```
Comment (christianh814) [inline templates/zonefile.j2:26]: I think setting this to a `CNAME` to the helper. Maybe to `{{ helper.name | lower }}.{{ dns.clusterid }}.{{ dns.domain | lower }}.` should be enough.

File: templates/reverse.j2 [hunk 2]
```diff
@@ -10,6 +10,12 @@ $TTL 1W
 ; syntax is "last octet" and the host must have fqdn with trailing dot
 1       IN      PTR     helper.{{ dns.clusterid }}.{{ dns.domain }}.
 
+{% if setup_registry.deploy %} 
+; Create entry for the local registry
+{{ helper.ipaddr.split('.')[3] }}	IN	PTR	registry.{{ dns.clusterid }}.{{ dns.domain | lower }}.
+;
+{% endif %}
+
```
Comment (christianh814) [inline templates/reverse.j2:18]: I don't think a PTR is needed. can you test without it?

Comment (cs-zhang) [reply templates/reverse.j2:18]: It is added for consist. If without it, the access to registry with FQDN works fine. Do you want to to remove it?

File: templates/zonefile.j2 (hunk 1 above)
Comment (cs-zhang) [reply templates/zonefile.j2:26]: Change to use helper.name is ok, but we may need to change several places where registry name is used. What do you think?

Comment (christianh814) [reply templates/zonefile.j2:26]: @cs-zhang Looks like this was already addressed in #94 ...so this part isn't needed yet.

File: templates/reverse.j2 (hunk 2 above)
Comment (christianh814) [reply templates/reverse.j2:18]: I would like to not have multiple PTR records (multiple A records and CNAMEs are fine though).

**Comment by christianh814** [comment]
Looks like #94 addressed some of the same things, so I'm closing this. If you find other issues please feel free to open up another PR. Thanks!

**Pull request closed without merging**